TASVideos.Common/Extensions/StringExtensions.cs

```diff
@@ -2,6 +2,41 @@
 
 namespace TASVideos.Extensions;
 
+public ref struct RegexMatchShim
+{
+#pragma warning disable SA1310
```

Comment:
No.  Why do this? Just name the variable ErrMsgZeroed and play nice with the naming conventions of this repository